distributions/aws/test/e2e/resources/custom-resource-templates/katib-experiment-random.yaml

```diff
@@ -52,6 +52,9 @@ spec:
       kind: Job
       spec:
         template:
+          metadata:
```

Comment:
what does this do?

Reply:
Apparently
> (Optional) Note: Katib’s experiments don’t work with Istio sidecar injection. If you are using Kubeflow with Istio, you have to disable sidecar injection. To do that, specify this annotation: sidecar.istio.io/inject: "false" in your experiment’s trial template.

https://www.kubeflow.org/docs/components/katib/hyperparameter/#example-using-random-search-algorithm